lib/octEncodeNormals.js

```diff
@@ -136,14 +137,18 @@ function patchTechnique(gltf, techniqueId) {
     }
 }
 
-function patchProgram(gltf, programId, normalVariableName) {
+function patchProgram(gltf, programId, normalVariableName, patchedShaders) {
     var programs = gltf.programs;
     var shaders = gltf.shaders;
     var program = programs[programId];
     var vertexShaderId = program.vertexShader;
+    if (patchedShaders[vertexShaderId]) {
```

Comment:
This should be `defined(patchedShaders[vertexShaderId])`. 

I know this works as-is, but `defined` is used throughout in this way to be more verbose.